### PR TITLE
product card

### PR DESCRIPTION
**Pull request by DenisGurskiy**
https://DenisGurskiy.github.io/layout_product-cards/
https://DenisGurskiy.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
+@import url(./styles/star.css);
```
Comment (Viktor-Kost) [inline src/style.css:1]: Consider using a relative path for importing stylesheets. For example, `@import url('./styles/star.css');`

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,56 @@
...
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="card__image"
+      >
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <div class="card__code">Product code: 195434</div>
+      <div class="wrapper card__wrapper">
+        <div class="star stars--4">
```
Comment (Viktor-Kost) [inline src/index.html:26]: Class name 'star stars--4' should be 'stars stars--4' to maintain BEM naming consistency

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,56 @@
...
+          <div class="star__stars"></div>
```
Comment (Viktor-Kost) [inline src/index.html:27]: Class name 'star__stars' should be 'stars__star' instead

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,56 @@
...
+
+          <div class="star__stars"></div>
+
+          <div class="star__stars"></div>
+
+          <div class="star__stars"></div>
```
Comment (Viktor-Kost) [inline src/index.html:33]: no needs to add an empty line between _oneline_ siblings, only with _multiline_ instead
```suggestion
          <div class="star__stars"></div>
          <div class="star__stars"></div>
          <div class="star__stars"></div>
```

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,56 @@
...
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="card__image"
+      >
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <div class="card__code">Product code: 195434</div>
```
Comment (Viktor-Kost) [inline src/index.html:24]: use more semantic tags for text elements
```suggestion
      <p class="card__code">Product code: 195434</p>
```

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,56 @@
...
+        </div>
+        <span class="wrapper__reviews">
+          Reviews: 5
+        </span>
+      </div>
+
+      <div class="price card__wrapper">
+        <span class="price__text">Price:</span>
+        <span class="price__number">$2,199</span>
+      </div>
```
Comment (Viktor-Kost) [inline src/index.html:45]: semantic
```suggestion
        <p class="wrapper__reviews">
          Reviews: 5
        </p>
      </div>

      <div class="price card__wrapper">
        <p class="price__text">Price:</p>
        <p class="price__number">$2,199</p>
      </div>
```

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,56 @@
...
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="card__image"
+      >
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <div class="card__code">Product code: 195434</div>
+      <div class="wrapper card__wrapper">
```
Comment (Viktor-Kost) [inline src/index.html:25]: you need only two BEM blocks instead (`card` and `stars`)

Comment (Viktor-Kost) [inline src/index.html:25]:
```suggestion
      <div class="card__rating">
```

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,56 @@
...
+
+          <div class="star__stars"></div>
+
+          <div class="star__stars"></div>
+
+          <div class="star__stars"></div>
+
+          <div class="star__stars"></div>
+        </div>
+        <span class="wrapper__reviews">
```
Comment (Viktor-Kost) [inline src/index.html:37]:
```suggestion
        <span class="card__reviews">
```

File: src/index.html (hunk 6 above)
Comment (Viktor-Kost) [inline src/index.html:45]:
```suggestion
      <div class="card__price-wrapper">
        <span class="card__price-text">Price:</span>
        <span class="card__price-amount">$2,199</span>
      </div>
```

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+.card__wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.wrapper__reviews {
+  font-size: 10px;
+  color: #060b35;
```
Comment (Viktor-Kost) [inline src/style.css:55]: move repeated colors to variables

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card__buy {
+  background: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  width: 100%;
+  height: 40px;
```
Comment (Viktor-Kost) [inline src/style.css:77]: it's better not hardcode height for elements with content inside. Just use padding instead

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
```
Comment (Viktor-Kost) [inline src/style.css:30]:
```suggestion
  margin-bottom: 4px;
```

File: src/styles/star.css [hunk 12]
```diff
@@ -0,0 +1,29 @@
+/* #region star*/
```
Comment (Viktor-Kost) [inline src/styles/star.css:1]: remove comments
```suggestion
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin-top: 0;
```
Comment (vlad-demchuk) [inline src/style.css:33]: you can reset margins for tag and then apply only one

**Review by vlad-demchuk — APPROVED** [review]

File: src/style.css (hunk 13 above)
Comment (Viktor-Kost) [reply src/style.css]: reset default margin at `p` tag:
```
p {
  margin: 0
}
```

**Review by Viktor-Kost — APPROVED** [review]

**Pull request closed without merging**